### DISCUSSION TITLE
Can I Install on current Ubuntu Full Node?

### DISCUSSION THREAD

**Discussion by aaron-morph** [madMAx43v3r/chia-plotter#167] [Q&A]
Can I run this install on my current full node farmer/plotter that is running on Ubuntu 20?
Do I need to make any config changes with my current install?

TIA

**Comment by Dartellum** [comment]
Yes, you can run this on your Full mode. No changes required.

**Reply by aaron-morph** [reply]
Thanks for the quick response. Can't wait to try it out
